turbo/execution/eth1/inserters.go

```diff
@@ -62,6 +63,8 @@ func (e *EthereumExecutionModule) InsertBlocks(ctx context.Context, req *executi
 			return nil, fmt.Errorf("parent's total difficulty not found with hash %x and height %d: %v", header.ParentHash, header.Number.Uint64()-1, err)
 		}
 
+		metrics.UpdateBlockProducerProductionDelay(header.Time, height-1, e.logger)
```

Comment:
I believe this may need to be `UpdateBlockConsumerHeaderDownloadDelay` + `UpdateBlockConsumerBodyDownloadDelay` instead?